Makefile

```diff
@@ -82,7 +82,19 @@ vagrant-deploy: vagrant-sync-config vagrant-sync-build
 .release-functest:
 	make functest > .release-functest 2>&1
 
+kubevirt.yaml: TAG=$(shell git describe) PREFIX=kubevirt
+kubevirt.yaml:
+	echo "docker_prefix=$(PREFIX)" > hack/config-local.sh
```

Comment:
Wouldn't that override config-local.sh files from developers?

Reply:
You catched it. Yes it would.